schemas/com.mbta.ctd.glides.trips_updated.v1.json

```diff
@@ -181,6 +190,9 @@
           "maxItems": 2,
           "items": { "$ref": "#/$defs/car" }
         },
+        "revenue": {
+          "type": "boolean"
```

Comment:
question: do you think we might want more trip types in the future, such that it would make more sense to make this an enum like `status` with values `revenue` and `nonrevenue`? It seemed plausible to me. e.g. what we did in the API: https://github.com/mbta/api/blob/f7b2efe4057f2261ab547c206289d32050061789/apps/api_web/lib/api_web/controllers/trip_controller.ex#L251-L260

Reply:
oh, hm, I hadn't considered that.

I think "is this vehicle in revenue service?" could really be a boolean answer rather than an enum answer with two known options. There are many other statuses, but they could be separate fields.

Brainstorming future statuses we could add:
- "disabled"? This is something we'd be likely to add, but both revenue and nonrevenue vehicles could become disabled, so that should be a separate field rather than 4 statuses "rev disabled"/"rev nondisabled"/"nonrev disabled"/"nonrev nondisabled".
- type of nonrevenue vehicle, eg. wire car / instruction car. This could be useful, but there are many things you could do just knowing that a trip is nonrevenue without knowing why, so we should have a revenue field you can reference without knowing the full list of nonrevenue vehicle types, and then a separate vehicle type field.
- "charter"? This really would be a 3rd state that's neither revenue nor nonrevenue, but MBTA doesn't do that.
- "unknown"? It would be a 3rd state, but since this data all comes from human entry, right now we always know. But I can imagine future features that would create unknown data (e.g. automatically detecting a departure, and then trying but failing to match it up to a trip on the trainsheet, but we want to publish some other data about the trip).

okay, I'm convinced, I'll switch it to an enum.

Reply:
Isn't "disabled" kind of a fuzzy concept though, since a lot of different things can go wrong with a vehicle? It seems sort of like that's a separate concern from other status values, and if a vehicle isn't taking passengers due to being disabled you'd just mark is as non-revenue.

That said, this is mostly just bikeshedding since it was brought up, but I agree with the overall idea of having an enum.

Reply:
Done!